CHANGELOG.asciidoc

```diff
@@ -60,6 +73,8 @@ https://github.com/elastic/beats/compare/v5.4.1...master[Check the HEAD diff]
 
 - Clean configured geoip.paths before attempting to open the database. {pull}4306[4306]
 - Fix `packetbeat.interface` options that contain underscores (e.g. `with_vlans` or `bpf_filter`). {pull}4378[4378]
+- Enable memcache filtering only if a port is specified in the config file. {issue}4335[4335]
+- Enable memcache filtering only if a port is specified in the config file. {issue}4335[4335]
```

Comment:
Can you cleanup the CHANGELOG, please? Unfortunately this happens often with backports.